src/main/java/org/folio/helper/PurchaseOrderHelper.java

```diff
@@ -512,6 +509,42 @@ private CompletableFuture<Void> setPoNumberIfMissing(CompositePurchaseOrder comp
     return completedFuture(null);
   }
 
+  public void updatePrefixAndSuffix(CompositePurchaseOrder poFromStorage, CompositePurchaseOrder updatedPo) {
+    String poNumber = "";
+    if(HelperUtils.isPrefixChanged(poFromStorage, updatedPo)) {
+      if(StringUtils.isNotEmpty(updatedPo.getPoNumberPrefix())) {
+        if(Objects.nonNull(poFromStorage.getPoNumberPrefix()) && updatedPo.getPoNumber().contains(poFromStorage.getPoNumberPrefix())) {
```

Comment:
please replace Objects.nonNull(poFromStorage.getPoNumberPrefix()) to StringUtils.isNotEmpty()